Tema3/src/Exercitiul3.java

```diff
@@ -0,0 +1,33 @@
+import java.text.DecimalFormat;
+import java.util.Scanner;
+
+/**
+ * Input Data Validation
+ * Write a program that asks the user what time it is, by printing on the console
+ * "What time is it?".Then the user must enter two numbers – one for hours and one for minutes.
+ * If the input data represents a valid time, the program must output the message " The time is hh:mm now.",
+ * where hh respectively means the hours, and mm – the minutes. If the entered hours or minutes are not valid,
+ * the program must print the message " Incorrect time!". The validation must be implemented in a separate method.
+ *
+ * @author Bogdan
+ */
+public class Exercitiul3 {
+
+    public static void main(String[] args) {
```

Comment:
again the idea was to have a separate method "The validation must be implemented in a separate method."